### PR TITLE
[new] add react-semantify@0.4.1

### PR DESCRIPTION
**Pull request by LeaYeh**
Hi @maruilian11  for #6671

I add react-semantify at v0.4.1 and add npm auto-update config in its package.json.

repo url: https://github.com/jessy1092/react-semantify
npm package url: https://www.npmjs.com/package/react-semantify

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : Watch 21, Star 198, Fork 19
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

File: ajax/libs/react-semantify/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "classnames": "^2.2.0",
+    "react": "^0.14.2",
+    "react-dom": "^0.14.2"
+  },
+  "npmName": "qwerty-hancock",
+  "npmFileMap": [
+    {
+      "basePath": "dst",
```
Comment (PeterDaveHello) [inline ajax/libs/react-semantify/package.json]: There is no `dst` folder.
```sh
$ ls qwerty-hancock/
bower.json  demo/  dist/  Gruntfile.js  karma-conf.js  lib/  package.json  README.md  src/  tests/
```

Comment (jessy1092) [inline ajax/libs/react-semantify/package.json]: @LeaYeh 
I think the npmName is `react-semantify`. https://www.npmjs.com/package/react-semantify

BTW. Thanks for adding `react-semantify` into cdnjs. :+1:

Comment (PeterDaveHello) [inline ajax/libs/react-semantify/package.json]: Thanks @jessy1092

Comment (LeaYeh) [inline ajax/libs/react-semantify/package.json]: @jessy1092 sorry for that, it's a typo error... thanks for your advice!

**Comment by LeaYeh** [comment]
@PeterDaveHello i fixed and rebased it :)

File: ajax/libs/react-semantify/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/react-semantify/package.json]: Nope that's not kind of typo ... anyway ... should be careful next time.

**Comment by PeterDaveHello** [comment]
@LeaYeh You should leave the second line of the commit message empty.

**Pull request merged**